### PR TITLE
Populate testgrid dashboardtab description from k8s.io/description annotation on prowjob

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: cmd/testgrid-config-generator/main.go [hunk 1]
```diff
@@ -278,7 +281,8 @@ func main() {
 				}
 			}
 
-			informing.add(p.Name, daysOfResults)
+			fmt.Printf("period job annotations: %s: %v\n", p.Name, p.Annotations)
```
Comment (smarterclayton) [inline cmd/testgrid-config-generator/main.go]: drop this, I think we're good to go.  let's check with sam and Jessica though on which annotation we use (I think it's k8s.io, I want to be positive)

Comment (bparees) [reply cmd/testgrid-config-generator/main.go]: whoops, debug. removed.

Comment (bparees) [reply cmd/testgrid-config-generator/main.go]: @spadgett says they only look at `openshift.io/description` and that's on projects.

Comment (smarterclayton) [reply cmd/testgrid-config-generator/main.go]: hrm.

**Comment by bparees** [comment]
@smarterclayton passing tests

**Pull request merged**